uuid/v1.ts

```diff
@@ -163,7 +167,7 @@ export function generate(
   b[i++] = clockseq & 0xff;
 
   for (let n = 0; n < 6; ++n) {
-    b[i + n] = node[n];
+    b[i + n] = node[n]!;
```

Comment:
Since we now validate the node length above we can be sure 0-5 will always be a valid value